src/packages/migrate/src/utils/seed.ts

```diff
@@ -97,6 +97,7 @@ runSeed()
   })
 `
 
+  // TODO Document why this is necessary for TS
```

Comment:
Because TypeScript throws errors if these extra are not passed.

We are actually considering removing that code and only execute as a script to simplify remove complexity and make things more reliable here.

Reply:
Interesting, so this is/was for working around that specific properties of TS.